src/app/models/evt-models.ts

```diff
@@ -127,6 +127,23 @@ export interface WitnessGroup {
     groupId: string;
 }
 
+export type ApparatusEntriesList = Map<ApparatusEntry>;
+
+export interface ApparatusEntry extends GenericElementData {
+    id: string;
+    content: ApparatusReading[];
+    note: NoteData[];
+    variance: number;
+    originalEncoding: string;
+}
+
+export interface ApparatusReading extends GenericElementData {
+    tagname: string;
+    id: string;
+    wits: string[];
```

Comment:
Shouldn't these be part of the lemma/rdg?

Reply:
Yes, if we distinguish them.

Reply:
https://www.google.com/url?sa=i&url=https%3A%2F%2Fgiphy.com%2Fexplore%2Fben-lucas&psig=AOvVaw1KL2QpPzsJUXSDea9OBxXY&ust=1585832606753000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJC3xPykx-gCFQAAAAAdAAAAABAD